core/services/vrf/v2/listener_v2_types.go

```diff
@@ -179,7 +179,9 @@ func (lsn *listenerV2) processBatch(
 		for _, reqID := range batch.reqIDs {
 			reqIDHashes = append(reqIDHashes, common.BytesToHash(reqID.Bytes()))
 		}
-		ethTX, err = lsn.txm.CreateTransaction(txmgr.TxRequest{
+		ctx, cancel := lsn.chStop.NewCtx()
+		defer cancel()
```

Comment:
Same question here :thinking: 
Maybe move this one to the top of the func, just to keep it simple and clear (and in scope for anything else to use easily).